### PR TITLE
add tags newsletter, noaa-fisheries, nasa-framework

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
This draft PR is a WIP with notes to self here

(edited to say I added this tag)
Do I add `newsletter` tag to the following posts?
"Update" in title:
  - https://www.openscapes.org/blog/2021/11/03/champions-fall-cohorts-2021/

**Comment by stefaniebutland** [comment]
I think I've tagged (most of?) the relevant posts with `newsletter` (incl searching for "update in title), `noaa-fisheries` and `nasa-framework`. For the noaa and nasa tags, I searched the blog for posts referring to the relevant terms and scanned for actual relevance.

File: content/blog/welcome-erin-2021-momentum.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ categories:
 tags:
   - impact
   - newsletter
+  - noaa-fisheries
```
Comment (jules32) [inline content/blog/welcome-erin-2021-momentum.md]: I think this one we don't need to tag with `noaa-fisheries`. But I am new enough to github reviews that I don't know how to reject this, I will try.

**Review by jules32 — CHANGES_REQUESTED** [review]
Hi @stefaniebutland ! This all looks great. My only request (and this also serves the purpose of practicing this "request changes" GitHub review feature) is to not tag the "welcome-erin-2021-momentum.md" post with the `noaa-fisheries` tag. Thank you!

File: content/blog/welcome-erin-2021-momentum.md (hunk 1 above)
Comment (stefaniebutland) [inline content/blog/welcome-erin-2021-momentum.md]:
```suggestion
```

**Comment by stefaniebutland** [comment]
I can screenshare to show you a cool review / suggest changes feature some time.

I've committed the change you requested. Should be good to merge now.

**Review by jules32 — APPROVED** [review]
thanks!

**Comment by jules32** [comment]
Merged! Thanks @stefaniebutland , i'll delete this branch too

**Pull request merged**